tools/king/king.xml

```diff
@@ -1,15 +1,77 @@
-
 <tool id="king" name="KING" version="2.2.4" python_template_version="3.5">
     <requirements>
-        <requirement type="package">king</requirement>
+	<requirement type="package">king</requirement>
```

Comment:
a package version is needed here. Do `conda search -c bioconda king` to see what the latest version is